Exercises_Java/src/Main.java

```diff
@@ -0,0 +1,25 @@
+public class Main {
+    public static void main(String[] args){
+        /*Put any age and discover your school Grade
+        * IF-ELSE
+        */
+        SchoolGrades grades= new SchoolGrades(32);
+        grades.knowingMyGrade();
+        System.out.println(grades.getAge());
+
+        System.out.println();
```

Comment:
Asumo que este print es solo para dar un salto de linea? te sugiero que en lugar de este print agregues el salto de linea en el texto que ya imprimes en la linea 8. Esto se hace agregando un "\n" en la parte que quieres dar el salto. Ex. 
System.out.println(grades.getAge() + "\n"); o System.out.println("Sorry!, You cannot be in school yet.\n");